keps/sig-api-machinery/1929-built-in-default/kep.yaml

```diff
@@ -0,0 +1,22 @@
+title: Built-in declarative defaults
+kep-number: 1929
+authors:
+  - apelisse
+owning-sig: sig-api-machinery
+status: provisional
```

Comment:
For this to be included in the 1.20 milestone it must be merged as implementable by October 6th.